bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogMetadata.java

```diff
@@ -96,4 +100,49 @@ public String toString() {
         return sb.toString();
     }
 
+    /**
+     * Serializes {@link EntryLogMetadata} and writes to
+     * {@link DataOutputStream}.
+     * 
+     * @param out
+     * @throws IOException
+     *             throws if it couldn't serialize metadata-fields
+     * @throws IllegalStateException
+     *             throws if it couldn't serialize ledger-map
+     */
+    public void serialize(DataOutputStream out) throws IOException, IllegalStateException {
+        out.writeLong(entryLogId);
```

Comment:
should we have some versioning flag ?

Reply:
good idea. it will be helpful in future if we want to add/change definition.